examples/stats/web_client_monitoring/web_client/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
```

Comment:
I think ```.gitignore``` and ```package-lock.json``` are not required to checkin in case of examples

Reply:
Removed those files in the latest commit.